source/opt/inline_pass.h

```diff
@@ -117,11 +134,31 @@ class InlinePass : public Pass {
                      ir::UptrVectorIterator<ir::Instruction> call_inst_itr,
                      ir::UptrVectorIterator<ir::BasicBlock> call_block_itr);
 
-  // Returns true if |inst| is a function call that can be inlined.
+  // Return true if |inst| is a function call that can be inlined.
   bool IsInlinableFunctionCall(const ir::Instruction* inst);
 
-  // Returns true if |func| is a function that can be inlined.
-  bool IsInlinableFunction(const ir::Function* func);
+  // Compute structured successors for function |func|.
+  // A block's structured successors are preceded by its
+  // merge block if its a header. This assures correct depth
+  // first search in the presence of early returns and kills.
+  void ComputeStructuredSuccessors(ir::Function* func);
+  
+  // Return function to return successors for a given block
+  GetBlocksFunction SuccessorsFunction();
```

Comment:
Should say that this assumes ComputedStructuredSuccessors has been called.
For otherwise the data is invalid.